openeo-geotrellis/src/test/scala/org/openeo/geotrellis/layers/FileLayerProviderTest.scala

```diff
@@ -804,8 +811,8 @@ class FileLayerProviderTest {
     assertEquals(0,minKey.col)
     assertEquals(0,minKey.row)
     assertEquals(crs,result._2.crs)
-    assertEquals(12,all.length)
-    //assertEquals((cols*rows).toInt,all.length)
+    assertTrue(1 <= all.length) // dependant on selectLayoutScheme
```

Comment:
not sure if we really want to relax the assertion? It was in place to be really sure that we get the right amount of tiles back.

Reply:
I added a more strict assertion just below. I left the very relax assertion just in case.